apps/style/common.scss

```diff
@@ -107,7 +107,7 @@ html[dir='rtl'] #visualizationColumn {
   }
 }
 
-#pause-button:focus:not(:focus-visible) {
+.accessibility-friendly-no-focus-outline:focus:not(:focus-visible) {
```

Comment:
Nit: Let's make this class name a little shorter to be more eng-friendly and then let's add a comment here to document the rest of the context.